### ISSUE TITLE
Update to designstart_r2p0

### ISSUE THREAD

**Issue by siorpaes** [rbarzic/arty-cm0-designstart#4]
Hello,

I am trying to synthesize the design with "make synt" but it seems that ARM DesignStart r1p0 is needed. The r1p0 package unfortunately is not longer available from ARM as only DesignStart r2p0 can be downloaded.

Do you have any plan to update the project to this latest version ?

Many thanks in advance

BR

David

**Comment by rbarzic** [comment]
Hi - I'm quite busy right now - I'll try to look into that but not before a few days I'm afraid

**Comment by siorpaes** [comment]
All right, there's absolutely no urge :)

Thank you very much

David

**Comment by SeanHoulihane** [comment]
Just in case you'd not noticed, there is a version of Cortex-M1 packaged for Vivado available now (similar license to M0). It comes with a simple ARTY example. M1 is the FPGA optimised version of M0.

**Comment by rbarzic** [comment]
Yes thank you - I got notification from ARM about that - But didn't have time to look into it

**Comment by siorpaes** [comment]
Thanks @SeanHoulihane .  Got this interesting info today from ARM newsletter. I will definitely give it a look!
